src/cryptoadvance/specter/services/service_encrypted_storage.py

```diff
@@ -110,7 +110,7 @@ def __init__(self, data_folder: str, user: User, disable_decrypt: bool = False):
                 "ServiceUnencryptedStorage needs to be initialized with disable_decrypt = True"
             )
         if disable_decrypt:
-            super().__init__(data_folder, encryption_key=None, disable_decrypt=True)
+            super().__init__(data_folder, user, disable_decrypt=True)
```

Comment:
In there any reason `disable_decrypt` is still as an argument  in the `def __init__` ?

Reply:
As mentioned, the inheritance-hirarchy is screwed up. Would be cool to fix it, but don't want to do that much here. This PR is part of the preparation for the spectrum-extension PR.

However, you mean `ServiceUnencryptedStorage.__init__()` ? I can see the point. Is it possible to change the constructor in a derived class in python? I seriously don't know but as an old java-developer, i freeze a little bit changing constructor arguments in derived classes.

If you meant `ServiceEncryptedStorage.__init__()` ... well ... that's due to the screwed up hirarchy.

Reply:
sorry for timprecise description. I meant `ServiceUnencryptedStorage.__init__()`

> However, you mean ServiceUnencryptedStorage.__init__() ? I can see the point. Is it possible to change the constructor in a derived class in python?

AFAIK, yes.

> As mentioned, the inheritance-hirarchy is screwed up. Would be cool to fix it, but don't want to do that much here. This PR is part of the preparation for the spectrum-extension PR.

ok.

Reply:
Ok, so keeping that for now.